_pages/docs/how-does-it-work.md

```diff
@@ -0,0 +1,109 @@
+merged, they are moved out of that collection and into the `$ARCHIVED-COLL`
+collection. The generated merged document will be in the `$CONTENT-COLL` and
+`$MERGED-COLL` collections. 
+
+## Matching
+
+The matching process begins with a document, which we'll call the "original"
+document. This document may be selected because it's just been inserted into the
+database, or because a process is cycling through all content.
+
+The original document contains a set of properties, each with values. The
+[match configuration][match-config] specifies what properties are to be used
+for matching. The heart of the matching process is using these property values
+to construct a query to be run against the other entity documents in the
+database. Each property that is involved in matching turns into a `cts:query`;
+these queries are bundled together and used to run a search to find potential
+matches.
+
+The assembled query has two parts: a match query and a boost query. The boost
+query consists of queries for all properties, with their respective weights. The
+match query is a subset of all the queries, chosen to ensure that any potential
+match that gets returned will have a score of at least the lowest threshold
+configured in the match options.
+
+The combined query will be run once, generating a score-ordered sequence of
+potential matches, each of which is labeled according to a threshold of match
+probability. A match response will look like this:
+
+    <results uri="/source/3/doc3.json" index="1" total="5" score="79" threshold="Definitive Match" action="merge">
+
+### Matching Algorithms
+
+The default query to look for documents with property values that match the
+original document is a `cts:element-value-query`, looking for the exact same
+value that the original document has. In some cases, you might want to provide
+more flexibility in defining what a match is. For those cases, you can specify
+an algorithm.
+
+A matching algorithm function takes the value(s) of a single property from the
+original document, along with the match configuration, and uses that to generate
+a query to find other documents that have relevant values for the same property.
+This function will be run once for each original document. This is a normal
+XQuery or SJS function, so it can make database queries, call out to third-party
+services, or do anything else needed to generate that query.
+
+To see an example of a matching algorithm function, see [zip.xqy][zip.xqy].
+
+## Merging
+
+Merging takes a set of documents and creates a new document to represent the
+combination. The structure of the document will match the originals, which are
+assumed to have been harmonized. The merge configuration controls how property
+values from the input documents are preserved in the new document.
+
+When two or more documents get merged, they are removed from the
+[`$CONTENT-COLL`][constants]. A new document is added to the
```

Comment:
If we are going to keep referring to these constants, at least spell them out all the way.

Reply:
Do you mean use the actual name of the collection, like "mdm-content" here? I was trying to avoid that for the same reason we avoid using magic strings in code. Better to go with the actual names?